lib/galaxy_test/driver/api.py

```diff
@@ -126,6 +126,7 @@ class ApiTestInteractor(BaseInteractor):
     def __init__(self, test_case):
         admin = getattr(test_case, "require_admin_user", False)
         test_user = TEST_USER if not admin else ADMIN_TEST_USER
+        self.verify = True
```

Comment:
I don't think that's needed.
```suggestion
```

Reply:
I wad trying to deal with the test that failed. 
 `AttributeError: 'GalaxyInteractorApi' object has no attribute 'verify'`
(https://github.com/galaxyproject/galaxy/pull/9612/checks?check_run_id=585645835#step:11:597)
Could you help me?

Reply:
You need to move the new lines 109-117 before the call to `__get_user_key()` method on line 105 (which in turn call `_get()`, which looks for `self.verify`)